release/README.md

```diff
@@ -10,6 +10,13 @@ project that need to create a new release. We expect the reader to be familiar
 the project itself, [git][git-docs], [GitHub][github-guides], and
 [semantic versioning](https://semver.org).
 
+## 0. Verify most recent googleapis SHA adheres to policy
+
+In order to ensure that our released version includes a SHA of the service proto
+definitions that is both current and stable ([googleapis-sha-update-policy]),
+make sure that the commit SHA was updated at least 5 days prior, but no more
+than 7 days prior (TODO(#13062): automate this update process).
```

Comment:
Uhhh I think this language is too strict. What if we notice that it has been 8 days since the googleapis update? If we ever wanted to release again, we would have to:

1. update the googleapis SHA to latest, and wait 5 days

OR

2. teach update googleapis to not use latest, but some specific SHA from 5-7 days ago

---

2 actually isn't so bad... If we have that, step 0 could be "update googleapis to a SHA from 7 days ago".

Thoughts?